Source/UnitTests/Common/FileUtilTest.cpp

```diff
@@ -24,7 +24,7 @@ class FileUtilTest : public testing::Test
     }
   }
 
-  void SetUp()
+  void SetUp() override
```

Comment:
The Setup function in FileSystemTest and the declaration in TMDReaderTest could also use an override specifier while you're at it.

Reply:
Done. The one in TMDReaderTest is a bit unusual. It is a pure virtual function that provides an implementation, leading to the somewhat odd-looking `void SetUp() override = 0;`.